src/Nethermind/Nethermind.TxPool/Collections/BlobTxDistinctSortedPool.cs

```diff
@@ -1,20 +1,94 @@
 // SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
 // SPDX-License-Identifier: LGPL-3.0-only
 
+using System.Collections.Concurrent;
 using System.Collections.Generic;
 using Nethermind.Core;
 using Nethermind.Core.Crypto;
+using Nethermind.Core.Extensions;
+using Nethermind.Crypto;
 using Nethermind.Logging;
 
 namespace Nethermind.TxPool.Collections;
 
 public class BlobTxDistinctSortedPool(int capacity, IComparer<Transaction> comparer, ILogManager logManager)
     : TxDistinctSortedPool(capacity, comparer, logManager)
 {
+    protected override string ShortPoolName => "BlobPool";
+
+    public ConcurrentDictionary<string, List<Hash256>> GetBlobIndex => _blobIndex;
+
+    private readonly ConcurrentDictionary<string, List<Hash256>> _blobIndex = new();
```

Comment:
se byte[] as key with Bytes.EqualityComparer as supplied comparer for the dictionary.